### PR TITLE
[PTT-275] Add receipt to payment_option

### PR DESCRIPTION
**Pull request by infantesimone**
#### List of Changes

- new _text_ field _receipt_ in _payment_option_;
- storing receipt in payment_option in _sendRT_ service.

#### Motivation and Context

The goal of this PR is to save the receipt as text in table _payment_option_, in service _sendRT_.
The receipt is an xml which is saved as a string, as follows:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<paSendRTReq>
    <idPA>88888888888</idPA>
    <>77777777777</idBrokerPA>
    <idStation>77777777777_1</idStation>
    <receipt>
        <receiptId>c110729d258c4ab1b765fe902aae41d6</receiptId>
        <noticeNumber>382202100000000101</noticeNumber>
        <fiscalCode>88888888888</fiscalCode>
        <outcome>OK</outcome>
        <creditorReferenceId>11111111112222222</creditorReferenceId>
        <paymentAmount>30.00</paymentAmount>
        <description>test</description>
        <companyName>company EC</companyName>
        <officeName>office EC</officeName>
        <debtor>
            <uniqueIdentifier>
                <entityUniqueIdentifierType>F</entityUniqueIdentifierType>
                <entityUniqueIdentifierValue>JHNDOE00A01F999N</entityUniqueIdentifierValue>
            </uniqueIdentifier>
            <fullName>John Doe</fullName>
            <streetName>street</streetName>
            <civicNumber>12</civicNumber>
            <postalCode>89020</postalCode>
            <city>city</city>
            <stateProvinceRegion>MI</stateProvinceRegion>
            <country>IT</country>
            <e-mail>john.doe@test.it</e-mail>
        </debtor>
        <transferList>
            <transfer>
                <idTransfer>1</idTransfer>
                <transferAmount>20.00</transferAmount>
                <fiscalCodePA>77777777777</fiscalCodePA>
                <IBAN>IT0000000000000000000000000</IBAN>
                <remittanceInformation>remittanceInformation1</remittanceInformation>
                <transferCategory>transferCategory</transferCategory>
            </transfer>
            <transfer>
                <idTransfer>2</idTransfer>
                <transferAmount>10.00</transferAmount>
                <fiscalCodePA>77777777778</fiscalCodePA>
                <IBAN>IT0000000000000000000000001</IBAN>
                <remittanceInformation>remittanceInformation2</remittanceInformation>
                <transferCategory>transferCategory</transferCategory>
            </transfer>
        </transferList>
        <idPSP>88888888888</idPSP>
        <pspFiscalCode>88888888888</pspFiscalCode>
        <pspPartitaIVA>88888888888</pspPartitaIVA>
        <PSPCompanyName>PSP name</PSPCompanyName>
        <idChannel>88888888888_01</idChannel>
        <channelDescription>app</channelDescription>
        <payer>
            <uniqueIdentifier>
                <entityUniqueIdentifierType>F</entityUniqueIdentifierType>
                <entityUniqueIdentifierValue>JHNDOE00A01F205N</entityUniqueIdentifierValue>
            </uniqueIdentifier>
            <fullName>John Doe</fullName>
            <streetName>street</streetName>
            <civicNumber>12</civicNumber>
            <postalCode>89020</postalCode>
            <city>city</city>
            <stateProvinceRegion>MI</stateProvinceRegion>
            <country>IT</country>
            <e-mail>john.doe@test.it</e-mail>
        </payer>
        <paymentMethod>creditCard</paymentMethod>
        <fee>2.00</fee>
        <paymentDateTime>2021-10-01T17:48:22</paymentDateTime>
        <applicationDate>2021-10-01</applicationDate>
        <transferDate>2021-10-02</transferDate>
        <metadata>
            <mapEntry>
                <key>keytest</key>
                <value>1</value>
            </mapEntry>
        </metadata>
    </receipt>
</paSendRTReq>
```
#### How Has This Been Tested?

Run application locally with docker and use scripts in resources folder.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: report/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>it.gov.pagopa.hubpa</groupId>
       <artifactId>qworker</artifactId>
-      <version>0.0.6</version>
+      <version>0.0.7</version>
```
Comment (GiovanniMancini) [inline report/pom.xml:30]: :-)

File: payments/src/main/java/it/gov/pagopa/hubpa/payments/entity/PaymentOptions.java [hunk 2]
```diff
@@ -61,8 +65,13 @@
     private String receiptId;
     @Column(name = "fee", nullable = true)
     private BigDecimal fee;
+    @Column(name = "receipt", nullable = true)
+    @Lob
+    @Type(type = "org.hibernate.type.TextType")
+    @Basic(fetch = FetchType.LAZY)
+    private String receipt;
```
Comment (GiovanniMancini) [inline payments/src/main/java/it/gov/pagopa/hubpa/payments/entity/PaymentOptions.java:72]: maybe we should add the same change in Data Definition Language

**Review by GiovanniMancini — APPROVED** [review]

**Review by pasqualespica — APPROVED** [review]
🚀

**Comment by GiovanniMancini** [comment]
@infantesimone @pasqualespica could you rebase with main?

**Pull request merged**